exonum/src/blockchain/schema.rs

```diff
@@ -183,13 +183,13 @@ where
 
     /// Returns peers that have to be recovered in case of process restart
     /// after abnormal termination.
-    pub(crate) fn peers_cache(&self) -> MapIndex<&T, PublicKey, Connect> {
+    pub(crate) fn peers_cache(&self) -> MapIndex<&T, PublicKey, Message<Connect>> {
         MapIndex::new(PEERS_CACHE, &self.view)
     }
 
     /// Returns consensus messages that have to be recovered in case of process restart
     /// after abnormal termination.
-    pub(crate) fn consensus_messages_cache(&self) -> ListIndex<&T, RawMessage> {
+    pub(crate) fn consensus_messages_cache(&self) -> ListIndex<&T, Protocol> {
```

Comment:
I have a feeling that it sounds semantically strange, that we store `Protocol` in the index called `consensus_messages_cache`, but not some kind of message. Not quite sure if these are good suggestions for naming, but what about calling it something like `ExonumMessage` or `CoreMessage`?

Reply:
This suggestion is probably outside of this pr, but if we need it, we can change it within pr.